src/ctapipe/tools/calculate_pixel_stats.py

```diff
@@ -186,20 +182,19 @@ def start(self):
                         "No faulty chunks found for telescope 'tel_id=%d'. Skipping second pass.",
                         tel_id,
                     )
-            # Add metadata to the aggregated statistics
-            aggregated_stats.meta["event_type"] = dl1_table["event_type"][0]
-            aggregated_stats.meta["input_column_name"] = self.input_column_name
+            # Cnstruct the output table name based on the event type and the selected column name
+            output_table_name = f"{EventType(dl1_table['event_type'][0]).name.lower()}_{self.input_column_name}"
```

Comment:
Any reason why we here use a composite table name instead of a group / table hierarchy?

Reply:
this is not really a table name, this is a part of the group/table hierarchy... 
the actual path to table will be e.g.

```
/dl1/monitoring/telescope/calibration/camera/pixel_statistics/flatfield_image/tel_001
```

Reply:
It's fine.

Just noting that you can also iterate easily if it's in groups by doing `f.root[path].f_walk_nodes("Table")` for example.